Update secrets-management.md (#6405)
umbraco/UmbracoDocs: umbraco-cloud/set-up/project-settings/secrets-management.md

It will then use a reference that only the managed identity of the environment has access to.

{% hint style="info" %}

> Secret Management is available on all Cloud plans. For starter plans there is however a limit of 5 secrets per environment.

Reviewer: An attempt to make this info even shorter and sweeter. What do you think? 🤞 

```suggestion
Starter Plans have a limit of 5 secrets per environment, whereas higher-tiered plans have no limit.
```
